src/components/ClockModule.js

```diff
@@ -0,0 +1,286 @@
+import React, { useCallback, useRef, useState } from 'react'
+import {
+  blockExplorerUrl,
+  Button,
+  ButtonBase,
+  GU,
+  IconCheck,
+  IconClock,
+  IconCross,
+  IconExternal,
+  textStyle,
+  Timer,
+  useTheme,
+} from '@aragon/ui'
+import { useWallet } from 'use-wallet'
+
+import HeaderModule from './Header/HeaderModule'
+import HeaderPopover from './Header/HeaderPopover'
+import { useCourtClock } from '../providers/CourtClock'
+import { useCourtConfig } from '../providers/CourtConfig'
+import { useHeartbeat } from '../hooks/useCourtContracts'
+
+import { formatDuration } from '../utils/date-utils'
+import { shortenAddress, getNetworkType } from '../lib/web3-utils'
+
+import logoSvg from '../assets/LogoAccent.svg'
+
+function ClockModule() {
+  const buttonRef = useRef()
+  const [opened, setOpened] = useState(false)
+
+  const theme = useTheme()
+  const wallet = useWallet()
+  const onHeartbeat = useHeartbeat()
+  const courtConfig = useCourtConfig()
+
+  const {
+    currentTermId,
+    currentTermEndDate,
+    isSynced,
+    neededTransitions,
+  } = useCourtClock()
+
+  const toggle = useCallback(() => setOpened(opened => !opened), [])
+
+  const handlePopoverClose = useCallback(() => {
+    setOpened(false)
+  }, [])
+
+  const handleOnClick = useCallback(async () => {
+    try {
+      const tx = await onHeartbeat(neededTransitions)
+      await tx.wait()
+    } catch (err) {
+      console.error(err)
+    }
+  }, [neededTransitions, onHeartbeat])
+
+  const IconSync = isSynced ? IconCheck : IconCross
+
+  return (
+    <div
+      ref={buttonRef}
+      tabIndex="0"
+      css={`
+        display: flex;
+        align-items: center;
+        justify-content: space-around;
+        outline: 0;
+      `}
+    >
+      <HeaderModule
+        icon={
+          <div
+            css={`
+              background-color: ${theme.background};
+              display: flex;
+              border-radius: 100px;
```

Comment:
Wondering: are we just trying to turn this into a circle? We should use `border-radius: 50%` instead if so :)